components/org.wso2telco.analytics.sparkUdf/src/main/java/org/wso2telco/analytics/sparkUdf/service/BillUpdaterService.java

```diff
@@ -1,89 +1,93 @@
 package org.wso2telco.analytics.sparkUdf.service;
 
+import java.math.BigDecimal;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
 import org.joda.time.LocalDate;
 import org.killbill.billing.client.KillBillClient;
 import org.killbill.billing.client.KillBillClientException;
 import org.killbill.billing.client.KillBillHttpClient;
 import org.killbill.billing.client.RequestOptions;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.killbill.billing.client.RequestOptions'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.wso2telco.com:9000/coding_rules#rule_key=squid%3AUselessImportCheck)